purchase_security/views/purchase_team_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_team_form" model="ir.ui.view">
+            <field name="name">purchase.team.form</field>
+            <field name="model">purchase.team</field>
+            <field name="arch" type="xml">
+                <form string="Purchase Team">
+                    <group name="team">
+                        <field name="name" />
+                        <field name="user_ids" />
```

Comment:
Maybe you can customize the tree a little (or use a different one) so we don't have much columns or you can copy from the `sales_team.crm_team_view_form` view that uses a kanban which is more visually pleasant.